### PR TITLE
Update kernel config for 5.10 kernel

### PR DESCRIPTION
**Pull request by bkothapx**
lts2020-yocto: Enable android base and recommended configs.

Tracked-On: OAM-104003
Signed-off-by: Jeevaka Prabu Badrappan <jeevaka.badrappan@intel.com>
Signed-off-by: Kothapeta, Bikshapathi <bikshapathix.kothapeta@intel.com>

### CODE REVIEW

**Review by kaushlen — APPROVED** [review]

**Review by rajucm2007 — APPROVED** [review]
LGTM

**Review by JeevakaPrabu — CHANGES_REQUESTED** [review]
Similar change already integrated into celadon/s/mr0/stable. Please test with latest build and check the need for this PR

File: groups/kernel/gmin64/config-lts/lts2020-yocto/x86_64_defconfig [hunk 1]
```diff
@@ -264,6 +267,7 @@ CONFIG_SLAB_FREELIST_RANDOM=y
 # CONFIG_SHUFFLE_PAGE_ALLOCATOR is not set
 CONFIG_SYSTEM_DATA_VERIFICATION=y
 CONFIG_PROFILING=y
+CONFIG_TRACEPOINTS=y
```
Comment (JeevakaPrabu) [inline groups/kernel/gmin64/config-lts/lts2020-yocto/x86_64_defconfig:270]: Check whether this change is needed

File: groups/kernel/gmin64/config-lts/lts2020-yocto/x86_64_defconfig [hunk 2]
```diff
@@ -7060,28 +7043,42 @@ CONFIG_HAVE_SYSCALL_TRACEPOINTS=y
 CONFIG_HAVE_FENTRY=y
 CONFIG_HAVE_OBJTOOL_MCOUNT=y
 CONFIG_HAVE_C_RECORDMCOUNT=y
+CONFIG_TRACE_CLOCK=y
```
Comment (JeevakaPrabu) [inline groups/kernel/gmin64/config-lts/lts2020-yocto/x86_64_defconfig:7046]: all the trace related configs needs to be double checked

**Review by JeevakaPrabu — CHANGES_REQUESTED** [review]

**Review by rajucm2007 — APPROVED** [review]
LKGM

File: groups/kernel/gmin64/config-lts/lts2020-yocto/x86_64_defconfig (hunk 2 above)
Comment (bkothapx) [reply groups/kernel/gmin64/config-lts/lts2020-yocto/x86_64_defconfig:7046]: Yes this is already available in 
CELADON/device/intel/mixins/groups/kernel/gmin64/config-lts/lts2021-chromium/x86_64_defconfig

File: groups/kernel/gmin64/config-lts/lts2020-yocto/x86_64_defconfig (hunk 1 above)
Comment (bkothapx) [reply groups/kernel/gmin64/config-lts/lts2020-yocto/x86_64_defconfig:270]: Yes this is already available in 
CELADON/device/intel/mixins/groups/kernel/gmin64/config-lts/lts2021-chromium/x86_64_defconfig

File: groups/kernel/gmin64/config-lts/lts2020-yocto/x86_64_defconfig [hunk 3]
```diff
@@ -6746,6 +6725,8 @@ CONFIG_SYSTEM_TRUSTED_KEYS=""
 # CONFIG_SYSTEM_BLACKLIST_KEYRING is not set
 # end of Certificates for signature checking
 
+CONFIG_BINARY_PRINTF=y
```
[the remark this answers is not in the document]
Comment (bkothapx) [reply groups/kernel/gmin64/config-lts/lts2020-yocto/x86_64_defconfig:6728]: Yes, this is already available in 
CELADON/device/intel/mixins/groups/kernel/gmin64/config-lts/lts2021-chromium/x86_64_defconfig

File: groups/kernel/gmin64/config-lts/lts2020-yocto/x86_64_defconfig [hunk 4]
```diff
@@ -6967,7 +6949,7 @@ CONFIG_CC_HAS_WORKING_NOSANITIZE_ADDRESS=y
 CONFIG_HAVE_ARCH_KFENCE=y
 CONFIG_KFENCE=y
 CONFIG_KFENCE_SAMPLE_INTERVAL=500
-CONFIG_KFENCE_NUM_OBJECTS=63
+CONFIG_KFENCE_NUM_OBJECTS=255
```
[the remark this answers is not in the document]
Comment (bkothapx) [reply groups/kernel/gmin64/config-lts/lts2020-yocto/x86_64_defconfig:6952]: KFENCE is expected with these values else we observe its failing in VTS test case

**Review by kaushlen — APPROVED** [review]

**Review by JeevakaPrabu — APPROVED** [review]

**Pull request merged**